### DISCUSSION TITLE
wire:navigate not aborting previous navigation when other link was clicked meanwhile

### DISCUSSION THREAD

**Discussion by pronode** [livewire/livewire#7464] [7. Bugs (archived)] [answered]
### Livewire version

v3.2.6

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Set slow connection in dev tools, like "Slow 3G".
2. Have multiple wire:navigate links on your page.
3. Click any link and then quickly click other link.
4. Livewire will load first page and then switch to the second page instead just aborting first request and loading second link. 

### Additional information

This causes bad UX. Clicking wire:navigate link should abort any pending wire:navigate requests so the user with slow connection doesn't experiance interface flickering.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment, marked as answer]
We're in the process of investigating all wire:navigate issues and I have tested out your scenario. I believe this has been fixed, see https://livewire.laravel.com/docs/navigate#javascript-hooks for how to use `event.preventDefault()` to cancel, so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
